components/example-notebook-servers/base/Dockerfile

```
@@ -99,7 +108,7 @@ ENV LANGUAGE en_US.UTF-8
 ENV LC_ALL en_US.UTF-8
 
 # s6 - copy scripts
-COPY --chown=${NB_USER}:users --chmod=755 s6/ /etc
+COPY --chown=${NB_USER}:${NB_GID} --chmod=755 s6/ /etc
 
 USER $NB_UID
```

```suggestion
USER $NB_UID:$NB_GID
```

This is what I decided to do in https://github.com/opendatahub-io/kubeflow/pull/348, from abundance of caution. Here the user already has primary group 0 set in the `useradd`, so it should be as if the :0 was specified there.

I'm not sure what

> Note that when specifying a group for the user, the user will have only the specified group membership. Any other configured group memberships will be ignored.
> (from https://docs.docker.com/reference/dockerfile/#user)

precisely implies, but since group 0 is all that's needed, it need not bother me/us here.

---

I am avoiding setting an explicit group in the `USER` so we (or those who depend on the image) can support multiple groups if they want in the future.